rmgpy/cantherm/molepro.py

```diff
@@ -66,14 +66,17 @@ def loadEnergy(self,frequencyScaleFactor=1.):
         E0=None
         if f12a:
             while line!='':
-                #first one is for radicals second is for non radicals
-                if 'RHF-UCCSD(T)-F12a energy' in line or 'CCSD(T)-F12a total energy  ' in line:
+                if ('RHF-UCCSD(T)-F12a energy' in line
+                    or 'RHF-RCCSD(T)-F12a energy' in line
+                    or 'CCSD(T)-F12a total energy  ' in line):
```

Comment:
Was the blank space left here on purpose (`'CCSD(T)-F12a total energy  '`)?

Reply:
I didn't change that part, but I am fairly certain that the space is not necessary.

Reply:
I see. Let's not change it then, unless we can confirm it's unnecessary.